source/Kepler-Notifications/Subscription.class.st

```diff
@@ -4,43 +4,63 @@ I'm a Subscription.
 I know what message to send to a subscriber each time the event notification system receives an event.
 "
 Class {
-	#name : #Subscription,
-	#superclass : #Object,
+	#name : 'Subscription',
+	#superclass : 'Object',
 	#instVars : [
 		'eventType',
 		'subscriber',
 		'message'
 	],
-	#category : #'Kepler-Notifications'
+	#category : 'Kepler-Notifications',
+	#package : 'Kepler-Notifications'
 }
 
-{ #category : #'instance creation' }
+{ #category : 'instance creation' }
 Subscription class >> of: aSubjectOfInterest to: anEventType toBeNotifiedSending: aMessage [
 
-	^ self new initializeOf: aSubjectOfInterest to: anEventType toBeNotifiedSending: aMessage 
+	AssertionCheckerBuilder new
+		checking: [ :asserter |
+			asserter
+				enforce: [ aMessage argumentCount < 2 ]
+				because: 'Selector should have zero or one argument at most' ];
+		buildAndCheck.
```

Comment:
```suggestion
	AssertionChecker
				enforce: [ aMessage argumentCount < 2 ]
				because: 'Selector should have zero or one argument at most' ]
```
when there's only one check no need to use the builder